server/db/models/Order.js

```diff
@@ -0,0 +1,40 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+
+const Order = db.define("order", {
+  userId: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  status: {
+    type: Sequelize.ENUM("CART", "ORDER"),
+    defaultValue: "CART",
+    allowNull: false,
+  },
+  total: {
```

Comment:
I might advise against keeping a total on the order (at least for orders that aren't yet completed), since you can always calculate it from the individual products in the cart. If you want to do something that changes the total (like a discount code), it might be better to have another column on line items for adjustments and discounts

The main issue with keeping a total on the order, is that the total needs to be updated any time the cart changes